source/manual/incident-fixing.html.md

```diff
@@ -0,0 +1,21 @@
+---
+owner_slack: "@ana.botto"
+title: Fix an Incident
+parent: "/manual.html"
+layout: manual_layout
+type: learn
+section: Incidents
+---
+
+When there is an incident in an application you are supporting, it is important to make a quick fix so that users can get back to use the application as usual.
+
+You should look into the logs on Sentry and Logit to see where the error is coming from.
+
```

Comment:
I'd combine a few of these paragraphs into a bulleted list, Sentry and Logit should be separate lines, checking the past history of the commit messages another